GOVERNANCE.md

```diff
@@ -1,19 +1,20 @@
-## Governance
+## BC Government Aries OCA Bundle Registry Governance Document
+
+### Purpose
+This Governance Document outlines how the OCA Bundle Registry is managed and governed. It defines the Roles and Processes needed to publish OCA bundles that are used by Hyperledger Aries Agents. These OCA bundles contain instructions to correctly process and display digital credentials.
 
-This OCA Bundle registry is managed by the [BC Government Digital Trust Group].
 The contents of the registry is maintained in the [registry repository] and
 managed through a pull request submission, review and approval process. The
 details of how pull requests are managed is outlined in this Governance section
 of the registry.
-
+ 
 [BC Government Digital Trust Group]: https://digital.gov.bc.ca/digital-trust/
-[registry repository]: https://github.com/bcgov/aries-oca-bundles
+[Registry Repository]: https://github.com/bcgov/aries-oca-bundles
 
 ### Roles
 
 - The **[BC Government Digital Trust Group]** is the governing authority over this
-repository. It approves and merges (or not) pull requests that
-result in changes to the registry and the repository.
+repository. This group consists of a team of individuals within the Digital Credentials Services of the BC Government Ministry of Citizen's Services. This list is managed in the [MAINTAINERS](./MAINTAINERS.md) file. 
```

Comment:
So the idea is to add the list of people involved with the Governing Authority in the MAINTAINERS file, in addition to the Editors, right?  Two groups will be listed?

Reply:
As our governance model evolves, we can adjust the governance document. But for now, we need to just state how things work, today. Today, essentially, the Editors are the ones making decisions on PRs and Merges without external direction. If external direction becomes "a thing", then we can update the document to describe the process.

Reply:
I missed this comment.  I think we have external direction now — per notes below.